### PR TITLE
New events

### PR DESCRIPTION
**Pull request by osipov-mit**
### Changes
- Add new events follow https://github.com/gear-tech/gear/pull/1000

### Bump version of @gear-js/api to 0.20.0

### CODE REVIEW

File: api/test/DebugMode.test.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ describe('DebugMode', () => {
   test('enable debug mode', async () => {
     debug.enable();
     const transactionData = await sendTransaction(debug.enabled, alice, 'DebugMode');
-    expect(transactionData).toBe(true);
+    expect(transactionData[0]).toBe(true);
```
Comment (Pinkdesu) [inline api/test/DebugMode.test.ts:27]: .toBeTruthy()

File: api/test/DebugMode.test.ts [hunk 2]
```diff
@@ -56,6 +56,6 @@ describe('DebugMode', () => {
   test('disable debug mode', async () => {
     debug.disable();
     const transactionData = await sendTransaction(debug.enabled, alice, 'DebugMode');
-    expect(transactionData).toBe(false);
+    expect(transactionData[0]).toBe(false);
```
Comment (Pinkdesu) [inline api/test/DebugMode.test.ts:59]: .toBeFalsy()

File: api/test/utilsFunctions.ts [hunk 3]
```diff
@@ -1,37 +1,81 @@
...
-      return event.data[0].reply.unwrap()[1].toNumber();
+export const checkInit = (api: GearApi, programId: string) => {
+  let unsubs = [];
+  let messageId = undefined;
+  const resultPromise = Promise.race([
+    new Promise((resolve) => {
+      unsubs.push(
+        api.gearEvents.subscribeToGearEvent('ProgramChanged', (event) => {
+          if (event.data.id.toHex() === programId) {
+            if (event.data.change.isActive) {
```
Comment (Pinkdesu) [inline api/test/utilsFunctions.ts:20]: Maybe combine conditions

File: api/test/utilsFunctions.ts [hunk 4]
```diff
@@ -1,37 +1,81 @@
...
+            }
+          }
+        }),
+      );
+    }),
+    new Promise((resolve) => {
+      unsubs.push(
+        api.gearEvents.subscribeToGearEvent('UserMessageSent', (event) => {
+          if (event.data.source.toHex() === programId) {
+            if (event.data.reply.unwrap()[0].toHex() === messageId && event.data.reply.unwrap()[1].toNumber() !== 0) {
```
Comment (Pinkdesu) [inline api/test/utilsFunctions.ts:31]: Maybe combine conditions

**Review by nikitayutanov — APPROVED** [review]

**Pull request merged**